components/twilio/actions/make-phone-call/make-phone-call.mjs

```diff
@@ -36,12 +36,17 @@ export default {
     const toParsed = phone(this.to);
     console.log(toParsed);
     if (!toParsed || !toParsed.phoneNumber) {
-      throw new Error(`Phone number ${this.to} couldn't be parsed as a valid number.`);
+      throw new Error(`Phone number ${this.to} could not be parsed as a valid number.`);
+    }
+
+    const fromParsed = phone(this.from);
+    if (!fromParsed || !fromParsed.phoneNumber) {
+      throw new Error(`Phone number ${this.from} could not be parsed as a valid number.`);
```

Comment:
```suggestion
      throw new Error(`Phone number ${this.from} couldn't be parsed as a valid number.`);
```

Reply:
Changed 'could not' to 'couldn't' for conciseness (Google's Developer docs style guide)